src/main/java/hudson/plugins/copyartifact/DownstreamBuildSelector.java

```diff
@@ -292,7 +306,9 @@ public AutoCompletionCandidates doAutoCompleteUpstreamProjectName(
                 @AncestorInPath AbstractProject<?,?> project
         ) {
             // Specified Item to allow to autocomplete folders (maybe confusing...).
-            return AutoCompletionCandidates.ofJobNames(Item.class, value, project, project.getParent());
+            return project == null
+                    ? AutoCompletionCandidates.ofJobNames(Item.class, value, null)

```

Comment:
If there is no context, just turn off completions (return `new AutoCompletionCandidates()`).